x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/find_cases.ts

```diff
@@ -845,6 +845,22 @@ export default ({ getService }: FtrProviderContext): void => {
           // Only security solution cases are being returned
           ensureSavedObjectIsAuthorized(res.cases, 1, ['securitySolutionFixture']);
         });
+
+        it('should respect the owner filter when using range queries', async () => {
```

Comment:
nit: `when filtering severity`

Reply:
Just changed it!

Reply:
I think that the test should be outside of the `range queries` describe block and create one case in `securitySolutionFixture` with severity `low` and one in `observabilityFixture` with severity `low`. What do you think?